features/admin.actions.v1/components/action-config-form.tsx

```diff
@@ -111,10 +111,18 @@ const ActionConfigForm: FunctionComponent<ActionConfigFormInterface> = ({
     } = useGetActionsByType(actionTypeApiPath);
 
     const {
-        data: RulesMeta
+        data: RuleExpressionsMetaData
     } = useGetRulesMeta(actionTypeApiPath);
 
-    const showRulesComponent: boolean = false;
+    // TODO: Remove this temporary boolean once the management API is ready.
+    const showRulesComponent: boolean = true;
+
+    // TODO: Use this function to get the rule value.
+    /* eslint-disable @typescript-eslint/no-unused-vars */
+    const handleGetRuleValue = () => {
```

Comment:
```suggestion
    const handleGetRuleValue = (): void => {
```

Reply:
This is no longer valid in the latest update